index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Microverse Portofolio project</title>
+</head>
+
+<body>
+    <header>
+        <a class="logo">Dorcas</a>
+        <nav class="Union">
+            <img src="Union.png">
+        </nav>
+    </header>
+    <main class="content">
+        <section class="subcontent">
+            <h1> I'm Dorcas Uwimbabazi<br>
+                 Glad to see <br>
+                 you! </h1>
+                 
+                 <p>
+                    I’m a software developer! I can help you<br>
+                    build a product , feature or website Look<br>
+                    through some of my work<br>
+                    and experience! If you like what you see and<br>
+                    have a project you need coded, don’t<br>
+                    hestiate to contact me.
+                 </p>
+                 
+                 <div class="social_media">
```

Comment:
- [ ] Your website should look exactly like the design (e.g. font, colors, images, text, margins) using the templates in Figma. Kindly work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![readme](https://user-images.githubusercontent.com/70483835/219320161-23ffaf1e-1bc7-44e6-b100-93d8118c412c.png) | ![readme2](https://user-images.githubusercontent.com/70483835/219320208-662e4f97-5851-4c2a-9ddc-30cd24b6d323.png)